types/html-to-text/lib/block-text-builder.d.ts

```diff
@@ -1,3 +1,20 @@
+/**
+ * types of options for BlockTextBuilder methods.  These replace previously positional options.
+ */
+
+export interface addInlineOptions { noWordTransform?: boolean; }
+export interface openBlockOptions { leadingLineBreaks?: number;
+    reservedLineLength?: number;
+    isPre?: boolean; }
+export type blockTransformer = (str: string) => string;
+export interface closeBlockOptions { trailingLineBreaks?: number; blockTransform?: blockTransformer; }
+export interface openTableCellOptions { maxColumnWidth?: number; }
+export interface closeTableCellOptions { colspan?: number; rowspan?: number; }
+export interface closeTableOptions { colSpacing?: number;
+    rowSpacing?: number;
+    leadingLineBreaks?: number;
+    trailingLineBreaks?: number; }
```

Comment:
Could you PascalCase these types? It's more idiomatic.

Reply:
Thanks for the review.

I have mixed feelings about casing since they are more like types than interfaces but DT wants interfaces.  @KillyMXI Do you have a preference?

Reply:
I use PascalCase for my [typedefs](https://github.com/html-to-text/node-html-to-text/blob/master/lib/typedefs.js) and camelCase for members.

I can't see style guide differences between types and interfaces. Maybe we have different style guides in mind? I don't think I can speak confidently about idiomatic TypeScript.

Reply:
Types == Interfaces in TS for all intents and purposes. Only minor differences in certain places. It's idiomatic for both to be PascalCase.